src/gordon_gcp/plugins/service/gdns_publisher.py

```diff
@@ -305,9 +287,13 @@ def _format_resource_record_changes(self, action, resource_record):
         changes_to_publish = self._format_resource_record_changes(
             action, resource_record)
         try:
-            change_id = await self._publish_changes(changes_to_publish,
-                                                    base_changes_url)
-        except exceptions.GCPHTTPConflictError as e:
+            change_id = await self._publish_changes(
```

Comment:
Might want to leave the formatting here as it was, for consistency.

Reply:
I would argue this is more consistent with the surrounding method calls (or at least the two I can see in this file diff)

Reply:
I took another look... They were already inconsistent.  Can you change the rest of the ones in this function too?

            change_id = await self._publish_changes(changes_to_publish,
                                                    base_changes_url)

        change_complete = await self._changes_published(change_id,
                                                        base_changes_url)

Reply:
I think that is outside the scope of this PR.

Reply:
I mean, I would call it leaving things better than you found them, honestly, but I won't hold up the PR because of it.